src/js/common/utils/helpers.js

```diff
@@ -125,7 +125,6 @@ export function focusElement(selectorOrElement, options) {
     : selectorOrElement;
 
   if (el) {
-    el.setAttribute('tabindex', '-1');
```

Comment:
Removing this breaks the focus management on the forms when you move between pages. Maybe we can check for a tab index of 0 and not set it then?

Reply:
Thanks for the context @jbalboni. Updated.